Modules/Microsoft365DSC/DSCResources/MSFT_EXORoleGroup/MSFT_EXORoleGroup.psm1

```diff
@@ -0,0 +1,452 @@
+function Get-TargetResource {
```

Comment:
The code style you are using does not match the style used within M365DSC. Specifically this one: "One Newline Before Braces" here (https://dsccommunity.org/styleguidelines/whitespace/)

Are you using Visual Studio Code and have the PowerShell Extension installed, you can then run the formatter by pressing Alt-Shift-F which will apply the formatting used by the project.

Reply:
for some reason it is exactly the VS Code PS extension removing the new lines before braces when runing the formatter.